### PR TITLE
Add support for investments product

### PR DESCRIPTION
**Pull request by maxdjohnson**

### CODE REVIEW

File: src/main/java/com/plaid/client/request/InvestmentsTransactionsGetRequest.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  public static class BaseOptions {
+    public List<String> accountIds;
+    public Integer count;
+    public Integer offset;
+
+    private BaseOptions() {
+    }
+  }
+
+  public static class Options extends BaseOptions {
```
Comment (pbernasconi) [inline src/main/java/com/plaid/client/request/InvestmentsTransactionsGetRequest.java]: I don't know java well enough to understand if this empty class is necessary?

File: src/main/java/com/plaid/client/response/InvestmentsTransactionsGetResponse.java [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    private String investmentTransactionId;
+    private String accountId;
+    private String securityId;
+    private String date;
+    private String name;
+    private Double quantity;
+    private Double amount;
+    private Double price;
+    private Double fees;
+    @SerializedName("class") private String transactionClass;
```
Comment (pbernasconi) [inline src/main/java/com/plaid/client/response/InvestmentsTransactionsGetResponse.java]: I think we're missing `subclass` here?

File: src/main/java/com/plaid/client/response/InvestmentsTransactionsGetResponse.java [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+    public Double getAmount() {
+      return amount;
+    }
+
+    public Double getPrice() {
+      return price;
+    }
+
+    public Double getFees() {
```
Comment (pbernasconi) [inline src/main/java/com/plaid/client/response/InvestmentsTransactionsGetResponse.java]: missing `getClass` and `getSubclass`?

**Comment by pbernasconi** [comment]
:gem: LGTM after fixing the missing subclass!

File: src/main/java/com/plaid/client/request/InvestmentsTransactionsGetRequest.java (hunk 1 above)
Comment (maxdjohnson) [reply src/main/java/com/plaid/client/request/InvestmentsTransactionsGetRequest.java]: me neither tbh - copied from https://github.com/plaid/plaid-java/blob/03ef882fc417ab729da308ab5098feea3d493574/src/main/java/com/plaid/client/request/TransactionsGetRequest.java#L78

File: src/main/java/com/plaid/client/response/InvestmentsTransactionsGetResponse.java (hunk 2 above)
Comment (maxdjohnson) [reply src/main/java/com/plaid/client/response/InvestmentsTransactionsGetResponse.java]: we're not shipping `subclass` - removed it for GA

File: src/main/java/com/plaid/client/response/InvestmentsTransactionsGetResponse.java (hunk 3 above)
Comment (maxdjohnson) [reply src/main/java/com/plaid/client/response/InvestmentsTransactionsGetResponse.java]: No subclass, changed `getClass` to `getTransactionClass` due to `getClass` having a special meaning in Java (it won't compile)

**Pull request merged**